jetty-core/jetty-io/src/main/java/org/eclipse/jetty/io/RetainableByteBuffer.java

```diff
@@ -384,27 +426,37 @@ public ByteBuffer getByteBuffer()
             {
                 case 0 -> RetainableByteBuffer.EMPTY.getByteBuffer();
                 case 1 -> _buffers.get(0).getByteBuffer();
-                default -> copyBuffer();
+                default ->
+                {
```

Comment:
The `default` block looks like a dupe of `copy()`, and it leaks the RBB.

Reply:
I don't think it leaks because it saves the result in _buffers to be released when the buffer is.  But it could probably use copy to build the buffer....

Reply:
I've put in a common impl